samples/SimpleServer/Program.cs

```diff
@@ -61,6 +61,8 @@ public HelloResource(string uri) : base(uri)
 
         public override CoapMessage Get(CoapMessage request)
         {
+            Console.WriteLine($"Got request: {request}");
```

Comment:
Not needed, but I added it to make it visible when the server received a request